src/Corax/Indexing/IndexWriter.IndexEntryBuilder.cs

```diff
@@ -89,6 +89,14 @@ public void WriteNull(int fieldId, string path)
                 ExactInsert(field, Constants.NullValueSlice);
         }
 
+        public void WriteNonExistingField(int fieldId, string path)
+        {
+            var field = GetField(fieldId, path);
+
+            if (field.ShouldIndex)
+                ExactInsert(field, Constants.NonExistingValueSlice);
```

Comment:
I still have some doubts that we do it via `ExactInsert()`. @maciejaszyk what do you think?

Reply:
OK